package.json

```diff
@@ -82,7 +82,7 @@
 		"schoolImport": "node ./scripts/schoolImport.js"
 	},
 	"dependencies": {
-		"@aws-sdk/lib-storage": "^3.50.0",
+		"@aws-sdk/lib-storage": "^3.100.0",
```

Comment:
It has effect on the current working solution. QA must be test the old files services before we merged it. 
I put it as to do to our board.

Reply:
No, we use it in new FileStorage!